protos/fpc/fpc.proto

```diff
@@ -66,3 +66,36 @@ message Credentials {
     // serialized attestation evidence as output by `AttestationToEvidence`, see `interfaces.attestation.md`
     bytes evidence = 3;
 }
+
+message ChaincodeRequest {
+    // function to be invoked on FPC chaincode
+    string function = 1;
+
+    // function arguments of above
+    repeated string arguments = 2;
+
+    // key to (aes128-gcm) encrypt ChaincodeResponse
+    bytes returnEncryptionKey = 3;
+}
+
+message EncryptedChaincodeRequest {
+    // an RSA-encryption of the serialization of ChaincodeRequest
+    bytes payload = 1;
+}
+
+message ChaincodeResponse {
+    // A status code that should follow the HTTP status codes.
+    int32 status = 1;
+
+    // A message associated with the response code.
+    string message = 2;
+
+    // A payload that can be used to include metadata with this response.
+    bytes payload = 3;
+}
+
+message EncryptedChaincodeResponse {
+    // an aes128-gcm encryption of the serialization of ChaincodeResponse
+    bytes payload = 1;
+}
+
```

Comment:
```suggestion
message ProposalResponse {
    EncryptedChaincodeResponse encrypted_response = 1;
    bytes signature = 3;
    bytes enclave_vk = 4;

    // TODO
    // read/writeset
}
```

maybe this as well as a replacement for https://github.com/hyperledger-labs/fabric-private-chaincode/blob/master/internal/utils/utils.go#L19

Reply:
yep, realized that also overnight that we need the r/w set and the signature.  For R/W-set (and actual response probably also best to re-use existing fabric messages ...

Reply:
btw: why would we send the enclave _key_ and not the enclave _id_ ?

Reply:
see revision where i use now fabric types as appropriate

Reply:
> btw: why would we send the enclave key and not the enclave id ?

we should send the enclave_id!!!